packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/DiagramEventProcessor.java

```diff
@@ -131,43 +122,6 @@ public ISubscriptionManager getSubscriptionManager() {
 
     @Override
     public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDescriptionSink, IRepresentationInput representationInput) {
-        if (representationInput instanceof LayoutDiagramInput layoutDiagramInput) {
-            if (layoutDiagramInput.id().equals(this.currentRevisionId)) {
```

Comment:
I do not find this test statement in the `#refresh`, we may have a regression here.

Reply:
Done.
Check at the moment we received the layout.